src/app/common/utils/system-notifications/system-notifications.enums.ts

```diff
@@ -0,0 +1,11 @@
+export enum NotificationType {
```

Comment:
I would place enums in files ending in `.constants.ts`. Besides enums we can export from them for example constants in the form of the `const` like:
```typescript
export const SOMETHING = 'some constant' as const;
```